telecine/build.gradle

```diff
@@ -27,7 +27,7 @@ android {
     applicationId 'com.jakewharton.telecine'
 
     minSdkVersion 21
-    targetSdkVersion 22
+    targetSdkVersion 25
```

Comment:
Ok, I didn't pay attention to WRITE_EXTERNAL_STORAGE and WRITE_SETTINGS permissions that can be cumbersome to request at runtime, I agree.
(on the other hand, SYSTEM_ALERT_WINDOW is always granted by the Play Store app, hence only with release build).
I'll remove it.